controllers/oauth2client_controller.go

```diff
@@ -160,7 +160,7 @@ func (r *OAuth2ClientReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 		}
 
 		if found {
-			if fetched.Owner != oauth2client.Name {
+			if fetched.Owner != fmt.Sprintf("%s/%s", oauth2client.Name, oauth2client.Namespace) {
```

Comment:
Shouldn't this be namespace/name?

Reply:
when I list the clients with the hydra REST API it's orderd `name/namespace`. Using hydra 1.2.2

Reply:
Right, appears to come from here: https://github.com/ory/hydra-maester/blob/208d1451dbf2787f80fcdf5707a62b64c6f8eb74/api/v1alpha1/oauth2client_types.go#L181